dealership/contracts.py

```diff
@@ -1,12 +1,67 @@
 class Contract(object):
-    pass
+
+    employee_discount = 0.1
+
+    def __init__(self, vehicle, customer):
+        self.customer = customer
+        self.vehicle = vehicle
 
 
 class BuyContract(Contract):
+
+    # The __init__ here is required because we are extending
+    # the init of the parent (Contract) class
+    # Contract.__init__(self, vehicle, customer) activates the
+    # attributes of the parent class
+
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        Contract.__init__(self, vehicle, customer)
+        self.monthly_payments = monthly_payments
+
+    def total_value(self):
+
+        interest_rates = {'Car': 1.07, 'Motorcycle': 1.03, 'Truck': 1.11}
+
```

Comment:
I like how you used a variables for the interest rates and lease multipliers. That's a helpful abstraction!